scripts/buyV1Covers.js

```diff
@@ -9,10 +10,18 @@ async function main() {
   console.log('Network config:', config.networks[network.name]);
 
   const [owner] = await ethers.getSigners();
-  const quotationData = await ethers.getContractAt(
-    'TestnetQuotationData',
-    '0xa513E6E4b8f2a923D98304ec87F64353C4D5C853',
-  );
+
+  // const quotationData = await ethers.getContractAt(
+  //   'TestnetQuotationData',
+  //   '0xa513E6E4b8f2a923D98304ec87F64353C4D5C853',
+  // );
+
+  /**
+   * For local testing we need to add the `addOldCover` function
+   * from the `TestnetQuotationData` contract to the `LegacyQuotationData` contract.
+   * Then we can use this script to add old v1 covers
+   */
+  const quotationData = await ethers.getContractAt('LegacyQuotationData', '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9');
```

Comment:
1. We cannot modify `LegacyQuotationData` as it's deployed and immutable. `TestnetQuotationData` contract inherits `LegacyQuotationData` and adds the corresponding function - please use it instead.
2. As it is now, this script will only work on mainnet and mainnet forks because of the hardcoded QD address. In what context are you using this script?

Reply:
I'm using it locally, the hardcoded address is the one from the local deployment. I found it easier to modify it this way for local deployment. The instructions are purely for local testing purposes.

Reply:
Updated the deploy script to use the Testnet contract version for QuotationData so we don't need this workaround anymore. 👍

Reply:
Thx for pointing it out! 🙌